pkg/cloudprivateipconfig/cloudprivateipconfig.go

```diff
@@ -0,0 +1,40 @@
+package cloudprivateipconfig
+
+import (
+	"errors"
+	"net"
+	"strings"
+)
+
+type IPFamily string
+
+const (
+	// IP Address Families
```

Comment:
nit: cloudprivateipconfig.go:12:2: comment on exported const IPv4 should be of the form "IPv4 ..."